### DISCUSSION TITLE
Multi-line Editing in Visual Line Mode

### DISCUSSION THREAD

**Discussion by tibitTW** [VSCodeVim/Vim#9099] [Q&A] [answered]
Is it a unique functionality that we can use visual line mode to select multiple lines then press `shift` + `i` to edit with multiple lines? It seems not to work in native Vim, but it works in VSCode Vim. No issues or bugs here, just curious.

some comparison screen records 

https://github.com/VSCodeVim/Vim/assets/39118448/58b9b4b5-5a65-4338-9a62-4544cac900b7

https://github.com/VSCodeVim/Vim/assets/39118448/15926b88-3f59-4bf4-88ac-ea79af4eeeef

**Comment by gdanielson** [comment, marked as answer]
This works in vim but the results are not seen until you exit Input Mode (usually by pressing the Escape key). Once exiting Visual Mode/Input Mode you should see the results on all lines selected in Visual Mode.

**Reply by olefSch** [reply]
Is there a solution to make it visible for nvim too?
